modules/virt-cloning-vm-snapshot.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * virt/backup_restore/virt-backup-restore-snapshots.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="virt-cloning-vm-snapshot_{context}"]
+= Cloning a snapshot to create a VM by using the web console
```

Comment:
Snapshot can't be cloned
VM can be cloned

Reply:
@ShellyKa13  and @jpeimer, I've renamed the topic/procedure to *Cloning a VM by using the web console* and updated the steps. 

Just as an aside, when I click *Clone* in the last step, I get an error message saying "An error occurred Error "Virtual Machine volume rootdisk does not support snapshots" for field "spec.Source"."